### PR TITLE
feat: Use seperate auth lambda database user (#718)

### PR DESCRIPTION
**Pull request by basilv**
For #718 (at least partially). Changes to existing API lambda user are planned to be kept in a separate PR.

### CODE REVIEW

File: infrastructure/frontend/cloudfront.tf [hunk 1]
```diff
@@ -36,9 +36,7 @@ resource "aws_cloudfront_distribution" "web_distribution" {
   price_class         = "PriceClass_100"
 
   viewer_certificate {
-    acm_certificate_arn = "${var.cloudfront_certificate_arn}"
```
Comment (basilv) [inline infrastructure/frontend/cloudfront.tf:39]: Temporary hack to deploy front-end

File: infrastructure/server/aurora-v2.tf [hunk 2]
```diff
@@ -167,6 +168,40 @@ resource "aws_secretsmanager_secret_version" "famdb_apicreds_secret_version" {
 EOF
 }
 
+# Create Auth Lambda DB User Credentials
+
+variable "famdb_auth_lambda_user" {
```
Comment (basilv) [inline infrastructure/server/aurora-v2.tf:173]: The lamdba function resource name was suffixed with the environment. Should we do that for all resources by default?

Comment (ConradBoydElliottGustafson) [reply infrastructure/server/aurora-v2.tf:173]: It was a recommendation on the "rename" ticket. Doing this for all resources seems like a lot of change for no benefit. Definitely don't want to recreate the DB with a different name (data restore in prod is hard).

**Review by ConradBoydElliottGustafson — APPROVED** [review]
Looks awesome. Thanks for fixing frontend for tools environment!

**Review by ianliuwk1019 — APPROVED** [review]

**Review by ConradBoydElliottGustafson — COMMENTED** [review]
I know this is merged already, but the tools_deployment.yml file still has the branch push on it. Just a quick cleanup required.

**Comment by basilv** [comment]
I'm planning to use it on my next branch and will clean it up then.

**Pull request merged**